src/utils.ts

```diff
@@ -506,3 +507,10 @@ export function isCloudEnvironment() {
 export function isRunningInWSL(): boolean {
   return !!process.env.WSL_DISTRO_NAME;
 }
+
+/**
+ * Generates a date that is 30 days from Date.now()
```

Comment:
```suggestion
 * Generates a date that is 30 days from Date.now() in UTC
```

Reply:
I don't think this is necessary-- JS dates don't have a timezone, they're just milliseconds since 1970/01/01 UTC. As I understand it, most methods on Date objects operate in the local timezone, while there are some methods that will convert / use UTC

Reply:
Good point